app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -57,11 +68,30 @@ protected void onCreate(Bundle savedInstanceState) {
         // Notify presenter to attach
         presenter.attach(this);
 
+        checkBoxEnableUrl.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            if(isChecked) {
+                checkBoxEnableUrl.setTextColor(Color.parseColor("#000000"));
```

Comment:
It's changing the base URL. It is acting on its own. This is untestable under current scenario. You can keep this, but it usually builds up and hard the architecture. Ideally, non even an if statement should be present in the View